src/main/java/com/commercetools/sync/sdk2/products/helpers/ProductBatchValidator.java

```diff
@@ -192,8 +191,7 @@ private void collectReferencedKeysInAttributes(
         getReferencedKeysWithReferenceTypeId(variantDraft, ProductTypeReferenceImpl.PRODUCT_TYPE));
```

Comment:
why not using the interface here:
```suggestion
        getReferencedKeysWithReferenceTypeId(variantDraft, ProductTypeReference.PRODUCT_TYPE));
```

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/928/commits/67914a7483999a27f7d4e91b491c1b02bf299be9